src/api/generate-csv/map-application-to-csv/helpers/map-month-string/index.test.ts

```diff
@@ -0,0 +1,33 @@
+import mapMonthString from '.';
+
+describe('api/generate-csv/map-application-to-csv/helpers/map-month-string', () => {
+  describe('when value is 0', () => {
+    it('should return a formatted string with `month`', () => {
+      const result = mapMonthString(0);
```

Comment:
Assertions covering:
* `` (Empty)
* ABC
* !"£

Reply:
We don't need to do this, typescript prevents anything other than a number from being passed to the function. If e.g a string is passed the code will not compile.